tensorflow/python/kernel_tests/data_structures/map_stage_op_test.py

```diff
@@ -624,5 +624,20 @@ def testNonScalarKeyUnorderedMap(self):
         sess.run(t, feed_dict={x: 1})
 
 
+  def testNonScalarKeyMapPeek(self):
+    with self.session():
```

Comment:
I don't think we need this `with self.session()`.  Can you remove if it still passes?